gitless/tests/test_e2e.py

```diff
@@ -63,6 +63,18 @@ def assert_not_in_repo(*cmds):
       gl.status, gl.diff, gl.commit, gl.branch, gl.merge, gl.fuse, gl.remote,
       gl.publish, gl.history)
 
+class TestTrackEmptyDir(TestEndToEnd):
```

Comment:
Missing tests:

- Track complex empty directories structures
- Remove placeholder files when files are added to empty versioned directories